.cirrus.yml

```diff
@@ -0,0 +1,7 @@
+container:
+  image: node:latest
+
+check_code_committed_task:
+  check_script:
+    - node scripts/check_code_committed.js "npm run build"
+    - node scripts/check_code_committed.js "npm run format"
```

Comment:
As the reorganization is tricky, let's target `master` instead and add the formatting later.
```suggestion
```